src/material/core/tokens/m2/mat/_stepper.scss

```diff
@@ -10,7 +10,12 @@ $prefix: (mat, stepper);
 // Tokens that can't be configured through Angular Material's current theming API,
 // but may be in a future version of the theming API.
 @function get-unthemable-tokens() {
-  @return ();
+  @return (
+    // Border radius for step header on focus
+    header-focus-state-shape: 0,
```

Comment:
I think this should be `state-layer`, not just `state` since it refers to the semi-transparent layer on top of the header.